person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] We can auto-generate the `id` of a person so we don't have to pass it in everytime we create a person. Then you can remove the `id` from the list of parameters.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = rand(1...1000)
    ...
  ```